src/bci_build/package.py

```diff
@@ -1202,12 +1209,14 @@ def _get_python_kwargs(
     package_name="python-3.10-image",
     support_level=SupportLevel.L3,
     is_latest=True,
+    supported_until=datetime.date(2023, 12, 31),
     **_get_python_kwargs("3.10", OsVersion.SP4),
 )
 
 PYTHON_3_11_SP5 = LanguageStackContainer(
     package_name="python-3.11-image",
     support_level=SupportLevel.TECHPREVIEW,
+    supported_until=datetime.date(2024, 12, 31),
```

Comment:
No, this is wrong. Python 3.11 will be the next long term supported python on sle15. Its the sle15 eol date which we do not know yet (and we should maybe use a global sle15_general_support_end variable for all these places)